### ISSUE TITLE
Is detr need a big batch_size and at least 10k dataset?

### ISSUE THREAD

**Issue by cyy21** [facebookresearch/detr#294]
i follow the minimum implemention of detr and use the pretrained resnet50 with frozen_bn, and i implement the loss and matcher, the loss and the matcher is tested, then i use 1 img and want to overfit it, the classnum is 1, after 3 epoch the model's output is all same, i print the match indices during training, and found the indices is all different(i think the indices for 1 img should be same), then i just set match indices to be the same, the result has no change, and i notice  #169 may meet the same problem. So, what do you think about the problem, Is detr need a big batch_size and at least 10k dataset?

**Comment by thibo73800** [comment]
On my side I did follow the same procedure (on a tensorflow implementation) and the model can overfit with just a few samples
